pkg/events/events.go

```diff
@@ -1387,7 +1387,7 @@ var Definitions = eventDefinitions{
 			ID32Bit: sys32setuid32,
 			Name:    "setuid",
 			Syscall: true,
-			Sets:    []string{"default", "syscalls", "proc", "proc_ids"},
+			Sets:    []string{"syscalls", "proc", "proc_ids"},
```

Comment:
Are the other setXXXid syscalls noisy? Maybe we should include them after all?

Reply:
`sudo nano /etc/shadow` generated 22 events with all setXXXid, so it seems ok. Including them.